src/index.js

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import App from './App';
+import {unregister} from './registerServiceWorker';
```

Comment:
Add a call of `unregister()` below `ReactDOM.render(...)`

Reply:
Oops... fixed that.